src/abstractdataframe/sort.jl

```diff
@@ -20,6 +20,14 @@ struct UserColOrdering{T<:ColumnIndex}
 end
 
 # This is exported, and lets a user define orderings for a particular column
+"""
+User defined order for a particular column.  Used in conjuction with sort!(df::DataFrame, cols).
```

Comment:
Please quote code in the text. Also it would be better if example were runnable. Finally if a function accepts some arguments they should be documented.

Reply:
You are referring to the kwargs of `order`, correct?  Are there any besides `rev`?  I was a bit confused by the source.

Reply:
Any by "runnable", do you mean that it should avoid undefined variables?

Reply:
In the body of `ordering(col_ord::UserColOrdering, lt::Function, by::Function, rev::Bool, order::Ordering)` you have a list of accepted kwargs.

By "runnable" I mean that when we enable docstring testing during CI (which we plan in the future) the code should run without error (as otherwise CI will fail). In this case it means that the use of undefined variable. The additional benefit of ensuring "runnability" is that the user can copy-paste the code and test what it does.